Package.swift

```diff
@@ -1,8 +1,9 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.5
 import PackageDescription
 
 let package = Package(
     name: "swift-service-discovery",
+    platforms: [.macOS("12.0")],
```

Comment:
Might be worth calling this out explicitly in the patch as a semver major change. While we're here, can you add the relevant platforms for iOS/tvOS/watchOS as well? We can't retroactively add them.